source/reference/minio-mc/mc-mirror.rst

```diff
@@ -191,44 +173,61 @@ Parameters
    
    Overwrites object(s) on the :mc-cmd:`~mc mirror TARGET`.
 
+   For example, object A exists on both the Source and the Target.
+   Object A on the source changes.
+   Running `mc mirror --overwrite` replaces the version on the target with the version from the Source.
+
 .. mc-cmd:: --preserve, a
    
 
-   Preserve file system attributes and bucket policy rules of the
-   :mc-cmd:`~mc mirror SOURCE` on the
-   :mc-cmd:`~mc mirror TARGET`.
+   Preserve file system attributes and bucket policy rules of the :mc-cmd:`~mc mirror SOURCE` on the :mc-cmd:`~mc mirror TARGET`.
 
 .. mc-cmd:: --region
    
 
-   Specify the ``string`` region when creating new bucket(s) on the
-   target. 
+   Specify the ``string`` region when creating new bucket(s) on the target. 
 
    Defaults to ``"us-east-1"``.
 
 .. mc-cmd:: --remove
    
 
-   Removes extraneous object(s) on the target. 
+   Removes object(s) on the Target that do not exist on the Source. 
+
+   For example, objects A, B, and C exist on Source.
+   Objects C, D, and E exist on Target.
+
+   When running `mc mirror --remove`, objects A and B copy to Target and objects D and E are removed from the Target.
+   C already exists on both, so does not change on either.
+
+   After the action, only objects A, B, and C exist on both teh Source and the Target.
+
+   Use the `--remove` flag to keep the same set of objects on both Source and Target.
 
 .. mc-cmd:: storage-class, sc
    
 
-   Set the storage class for the new object(s) on the 
-   :mc-cmd:`~mc mirror TARGET`. 
+   Set the storage class for the new object(s) on the :mc-cmd:`~mc mirror TARGET`. 
          
-   See the Amazon documentation on
-   :aws-docs:`Storage Classes <AmazonS3/latest/dev/storage-class-intro.html>` 
-   for more information on S3 storage classses.
+   See the Amazon documentation on :aws-docs:`Storage Classes <AmazonS3/latest/dev/storage-class-intro.html>` for more information on S3 storage classses.
 
 .. mc-cmd:: --watch, w
    
 
-   Continuously monitor the :mc-cmd:`~mc mirror SOURCE` path and
-   synchronize changes.
+   Continuously monitor the :mc-cmd:`~mc mirror SOURCE` path and synchronize `PUT` and `DELETE` changes from Source to Target.
 
    Defaults to ``0`` (all objects).
 
+   For example, object A and B exist on the watched Source.
+   Objects A, B, and C exist on the watched Target.
+   
+   An action adds object D to Source and removes object B.
```

Comment:
"An action" feels too generic. A client is going to perform a PUT write op. If it's a local filesystem it would be a write operation.

Maybe "A client writes object D to the source"